libs/permit-utils/src/lib/getPermitUtilsInstance.ts

```diff
@@ -14,11 +14,11 @@ const CHAIN_UTILS_CACHE = new Map<number, Eip2612PermitUtils>()
  */
 const PROVIDER_UTILS_CACHE = new Map<string, Eip2612PermitUtils>()
 
-export function getPermitUtilsInstance(
+export async function getPermitUtilsInstance(
```

Comment:
it's a breaking change, `@cowprotocol/permit-utils` should be published with new major version